src/app/wallets/update-on-chain-receipt.ts

```diff
@@ -257,7 +257,7 @@ const processTxForHotWallet = async ({
 
   const ledger = LedgerService()
 
-  const displayPriceRatio = await getCurrentPriceAsPriceRatio({
+  const displayPriceRatio = await getCurrentPriceAsWalletPriceRatio({
```

Comment:
same here should be `getCurrentPriceAsDisplayPriceRatio`

Reply:
https://github.com/GaloyMoney/galoy/pull/2334#discussion_r1108847733